### PR TITLE
[SQL Lab] Show tooltip on schema and table select options

### LINKED ISSUE #7119: [SQL Lab] Show tooltip on schema and table select options
This is the improve the UX when the user has tables with long names with the same prefix

### PR DESCRIPTION
**Pull request by enricoberti**
Closes #7119

### CODE REVIEW

**Comment by mistercrunch** [comment]
Can you screenshot how it looks when hovered?

File: superset/assets/src/components/TableSelector.jsx [hunk 1]
```diff
@@ -127,6 +127,10 @@ export default class TableSelector extends React.PureComponent {
        return SupersetClient.get({ endpoint })
         .then(({ json }) => {
           const filterOptions = createFilterOptions({ options: json.options });
+          json.options.forEach((option) => {
+            // eslint-disable-next-line no-param-reassign
```
Comment (xtinec) [inline superset/assets/src/components/TableSelector.jsx]: Create a new objection from `options` and add the `title` fields instead to avoid the warning?

**Comment by enricoberti** [comment]
@mistercrunch 
> Can you screenshot how it looks when hovered?

It's the standard non-punch-in-your-face browser title
Screenshot 2019-03-26 9 44 33

File: superset/assets/src/components/TableSelector.jsx (hunk 1 above)
Comment (enricoberti) [reply superset/assets/src/components/TableSelector.jsx]: True that, I updated the commit!

File: superset/assets/src/components/TableSelector.jsx [hunk 2]
```diff
@@ -127,10 +127,15 @@ export default class TableSelector extends React.PureComponent {
...
+          const options = json.options.map(o => ({
+            value: o.value,
+            label: o.label,
+            title: o.label,
+          }));
           this.setState(() => ({
             filterOptions,
             tableLoading: false,
-            tableOptions: json.options,
+            tableOptions: options,
```
Comment (kristw) [inline superset/assets/src/components/TableSelector.jsx]: nit: can move the `map` block above here and avoid declaring `const options`

**Review by kristw — APPROVED** [review]
One minor comment

File: superset/assets/src/components/TableSelector.jsx (hunk 2 above)
Comment (enricoberti) [reply superset/assets/src/components/TableSelector.jsx]: thanks @kristw! i updated the code getting rid of the const. I initially did it verbose following the pattern of the previous variable

**Comment by mistercrunch** [comment]
LGTM

**Pull request merged**